src/transports/ssh.c

```diff
@@ -20,7 +20,9 @@
 
 #define OWNING_SUBTRANSPORT(s) ((ssh_subtransport *)(s)->parent.subtransport)
 
-static const char prefix_ssh[] = "ssh://";
+static const char * ssh_prefixes[] = { "ssh://", "ssh+git://", "git+ssh://" };
+#define SSH_PREFIX_COUNT (sizeof(ssh_prefixes) / sizeof(ssh_prefixes[0]))
```

Comment:
We have an `ARRAY_SIZE` macro (defined in `util.h`) that will do this for you.  We would typically drop this `define` entirely and use `ARRAY_SIZE()` in the `for` loop itself.

If you're dropping that, could you rewrite the line above as `char *ssh_prefixes` (without the space between the `*` and `ssh_prefixes`).